lib/fastlane/plugin/android_emulator/actions/android_emulator_action.rb

```diff
@@ -4,7 +4,6 @@
 module Fastlane
   module Actions
     module SharedValues
-      ANDROID_SDK_DIR = :ANDROID_SDK_DIR
```

Comment:
Did it work for you, last time I tried, I had problems with the default ENV

Reply:
Looking at other Fastlane plugins, I think SharedValues should only be declared in the plugin that initialises the constant — in this case it would be the update_sdk plugin.

Reply:
Ah now I remember.

The problem is, when you use the `emulator_plugin` without the `sdk_plugin`
Therefore the SharedValues is not registered and fails.

Can you please test that ;-)

Reply:
Maybe it makes more sense now to use an ENV like ANDROID_HOME, or ANDROID_SDK as Screengrab uses it already, than define a SharedValue is so restrictive

Reply:
The official variable is ANDROID_SDK_ROOT so Screengrab is a bit wrong here... ANDROID_HOME is the legacy version, and it's fine to check that one (see https://developer.android.com/studio/command-line/variables.html).

I'm not sure if environment variables set within a fastlane plugin will be exported outside of the current shell to following plugins... If it works, then I think it would be the best way -- but otherwise I would suggest my second pull request for the sdk update plugin: https://github.com/NovaTecConsulting/fastlane-plugin-android_sdk_update/pull/4

This lets us call a quick plugin in the before_all section which then provides the path in the lane_context for all future actions.

Reply:
Noted, let's get on one pull-request at a time.
I would propose to leave the SharedValue for now as it is (so it works without the android-sdk)

Reply:
A quick test revealed that modifying an environment variable from within a plugin *does* result in that modification being visible to the rest of the fastlane process (including to other plugins), but the modification is *not* retained _outside_ of the fastlane process itself.

So I'd recommend avoiding the SharedValue completely and going for the recommended ANDROID_SDK_ROOT environment variable.

We could change the sdk update plugin to set that variable, and then pick it up from the emulator plugin.

What do you think?

Having duplicate declarations of a SharedValue constant result in warnings every time the plugins are included together -- not particularly pretty...